packages/http/src/validator/deserializers/style/__tests__/label.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { HttpParamStyles } from '@stoplight/types';
+
+import { LabelStyleDeserializer } from '../label';
+import * as createObjectFromKeyValListModule from '../utils';
+
+describe('LabelStyleDeserializer', () => {
+  const labelStyleDeserializer = new LabelStyleDeserializer();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Is this needed? We're not mocking anything in this test file.

Reply:
Definitely not, I also removed it from the rest of deserializer tests. Good catch & bit smaller codebase.